MAUI/Rotator/Getting-started.md

```diff
@@ -0,0 +1,574 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Rotator control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Rotator (SfRotator) control, its elements and more.
+platform: maui 
+control: Rotator
+documentation: ug
+---
+
+# Getting Started with .NET MAUI Rotator (SfRotator)
+
+This section explains how to add the Rotator control and bind data to it. This section covers only the basic features needed to get started with the Syncfusion `.NET MAUI Rotator` control.
+
+## Adding a .NET MAUI Rotator reference
+
+The Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add `.NET MAUI Rotator` to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.Rotator`, and install it.
+
+## Handler registration 
+
+In the MauiProgram.cs file, register the handler for the Syncfusion core.
+
+{% highlight c# hl_lines="6 17" %}
+
+    using Microsoft.Maui;
+    using Microsoft.Maui.Hosting;
+    using Microsoft.Maui.Controls.Compatibility;
+    using Microsoft.Maui.Controls.Hosting;
+    using Microsoft.Maui.Controls.Xaml;
+    using Syncfusion.Maui.Core.Hosting;
+
+    namespace Rotator
+    {
+        public static class MauiProgram
+        {
+            public static MauiApp CreateMauiApp()
+            {
+                var builder = MauiApp.CreateBuilder();
+                builder
+                .UseMauiApp<App>()
+                .ConfigureSyncfusionCore()
+                .ConfigureFonts(fonts =>
+                {
+                    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                });
+
+                return builder.Build();
+            }      
+        }
+    }   
+
+{% endhighlight %} 
+
+## Adding the .NET MAUI Rotator control
+
+Step 1: Add the NuGet to the project as discussed in the above reference section. 
+
+Step 2: Add the namespace as shown in the following code sample.
+
+{% tabs %}
+{% highlight xaml %}
+
+	xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Rotator;assembly=Syncfusion.Maui.Rotator"
+
+{% endhighlight %}
+{% highlight c# %}
+
+	using Syncfusion.Maui.Rotator;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Create a Simple SfRotator 
+
+The `SfRotator` control is configured entirely in C# code or by using XAML markup. The following steps explain on how to create a `SfRotator` and configure its elements,
+
+* Adding namespace for the added assemblies. 
```

Comment:
2nd time explaining about namespace. Is this needed?

Reply:
I had remove the namespace explaination in Create a Simple SfRotator